tests/lsdb/nested/test_nestedframe.py

```diff
@@ -54,6 +54,16 @@ def test_nested_columns(test_dataset):
     assert test_dataset.nested_columns == ["nested"]
 
 
+def test_exploded_columns(test_dataset):
+    base_columns = test_dataset._meta.base_columns
+    assert base_columns == ["a", "b"]
+    subcolumns = test_dataset._meta.get_subcolumns()
+    assert test_dataset.nested_columns == ["nested"]
+    assert test_dataset["nested"].nest.columns == ["t", "flux", "band"]
+    assert subcolumns == ["nested.t", "nested.flux", "nested.band"]
+    assert test_dataset.exploded_columns == base_columns + subcolumns
```

Comment:
Ah! That's not the answer i want, then! When I talked to Kostya about this last wee, he said he would like to be able to specify "nested" as a default column, and that would pull in the entirety of column "nested". So maybe exploded should be `base_columns + nested_columns + subcolumns`?